### PR TITLE
Add base UG template to work off of

### PR DESCRIPTION
**Pull request by Aak242**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,41 +2,142 @@
...
 * Add todo `todo n/TODO_NAME d/DEADLINE`
+* Move to next week or month `next`
+* Move to previous week or month `prev`
+* Update task description `update <day> <taskIndex> <newDescription>`
+* Add new task `add <day> <taskType> <taskDescription>`
+* Delete task `delete <day> <taskIndex>`
+* Mark task as complete or not complete `mark <day> <taskIndex>`
+* Set priority level for task `priority <day> <taskIndex> <priority>`
+* Switch to month view `month`
+* Switch to week view `week`
```
Comment (NicholasTanYY) [inline docs/UserGuide.md:142]: good idea to include a cheatsheet!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -2,41 +2,142 @@
...
+### Setting Priority Level for a Task: `priority`
+
+Sets priority level for a task.
+
+Format: `priority <day> <taskIndex> <priority>`
+
+Example of usage:
+
+`priority Friday 1 High`
+
```
Comment (NicholasTanYY) [inline docs/UserGuide.md:98]: we will change this to reflect our implementations more accurately

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,42 +1,154 @@
...
-{Give steps to get started quickly}
+To quickly get started with CLI-nton, follow these simple steps:
 
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+1. Ensure that you have Java 11 or above installed on your system.
+2. Download the latest version of `CLI-nton` from [here](http://link.to/clinton).
+3. Open a command terminal and navigate to the directory where you downloaded CLI-nton.
+4. Run the application using the command `java -jar clinton.jar`.
+5. You're all set to start managing your tasks efficiently!
```
Comment (Mahesh1772) [inline docs/UserGuide.md:15]: Good job listing out the various steps required to run the program.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,42 +1,154 @@
...
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+### Moving to Previous Week or Month: `prev`
+
+Moves to the previous week or month view.
+
+Example of usage:
+
+```
+prev
+```
```
Comment (Mahesh1772) [inline docs/UserGuide.md:40]: Good template for us to build upon!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1,42 +1,154 @@
...
+
+### Quitting the Application: `quit`
+
+Exits the calendar application.
+
+Example of usage:
+
+```
+quit
```
Comment (Mahesh1772) [inline docs/UserGuide.md:129]: Nice catch adding the quit statement!

**Review by Mahesh1772 — COMMENTED** [review]
LGTM!

**Pull request merged**